lib/tail.js

```diff
@@ -23,7 +30,7 @@ function Tail(path, opts) {
       this.emit('line', str);
     });
   } else {
-    const tail = childProcess.spawn('tail', ['-n', options.buffer, '-F'].concat(path));
+    const tail = childProcess.spawn('tail', ['-n', options.buffer, followOpt].concat(path));
```

Comment:
I think it will be better to define variable here instead of top of the file.
```suggestion
    let followOpt = '-F';
    if (process.platform === 'openbsd'){
      followOpt = '-f';
    }

    const tail = childProcess.spawn('tail', ['-n', options.buffer, followOpt].concat(path));
```